Test/Program.cs

```diff
@@ -117,9 +119,47 @@ private static void PrintFileInfo(string file, string outputDirectory, Options o
 
                 // Try to find the overlay header
                 if (!WiseOverlayHeader.FindOverlayHeader(stream, options.Debug, out var header) || header == null)
-                {
-                    _statistics.AddInvalidPath(file);
-                    Console.WriteLine($"No valid header could be found in {file}, skipping...");
+                { 
+                    stream.Seek(0, SeekOrigin.Begin);
+                    IWrapper? pe = WrapperFactory2.CreateExecutableWrapper(stream);
```

Comment:
Future note:` FindOverlayHeader` should take the executable wrappers directly instead of trying to build it internally. This will reduce code duplication across a couple paths.